src/execution/index/art/iterator.cpp

```diff
@@ -95,125 +95,137 @@ bool Iterator::Scan(const ARTKey &upper_bound, const idx_t max_count, set<row_t>
 }
 
 void Iterator::FindMinimum(const Node &node) {
-	D_ASSERT(node.HasMetadata());
-
-	// Found the minimum.
-	if (node.IsAnyLeaf()) {
-		last_leaf = node;
-		return;
-	}
+	Node node_copy = node;
```

Comment:
Do we still need a copy here because we pass in a `const Node`? I wonder where in the `FindMinimum` we expect const-ness 🤔